### PR TITLE
Braden Mason PR

### PR DESCRIPTION
**Pull request by BradenEM**

### CODE REVIEW

File: todo/src/components/List.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  clearForm = e => {
+    e.preventDefault();
+    this.setState({ newTask: "" });
+  };
+
+  comboFunction = e => {
+    this.addTask(e);
+    this.clearForm(e);
+  };
```
Comment (Alex-AK) [inline todo/src/components/List.js]: This is a good idea, but not really necessary. It's pretty common to clear a form or input on submit. So you can combine both the addTodo and setState back to "" into a single addTask function.

File: todo/src/components/List.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+              onChange={this.handleChange}
+            />
+            <button onClick={this.comboFunction}>ADD</button>
+          </form>
+        </div>
+        <div>
+          {this.props.todos.map((todo, index) => (
+            <h3 onClick={e => this.toggleCompleted(index)} key={index}>
+              {todo.task} <span> </span>
+              {todo.completed && <i className="fas fa-check-double" />}
```
Comment (Alex-AK) [inline todo/src/components/List.js:42]: Well implemented.

File: todo/src/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import { Provider } from "react-redux";
+import { createStore } from "redux";
+import reducer from "./reducers/reducer";
+
+const store = createStore(reducer);
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
```
Comment (Alex-AK) [inline todo/src/index.js:14]: This file is well implemented.

File: todo/src/reducers/reducer.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { ADD_TODO, TOGGLE_COMPLETED } from "../actions/actions";
+
+const initialState = {
+  todos: [
+    {
+      task: "Test",
+      completed: false
+    }
+  ]
+};
```
Comment (Alex-AK) [inline todo/src/reducers/reducer.js:10]: Good initial state.

File: todo/src/reducers/reducer.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+function reducer(state = initialState, action) {
```
Comment (Alex-AK) [inline todo/src/reducers/reducer.js:12]: Good reducer.

**Review by Alex-AK — APPROVED** [review]
Code is well implemented. Check comments for specific code feedback.

File: todo/src/components/List.js (hunk 1 above)
Comment (BradenEM) [reply todo/src/components/List.js]: Noted

**Pull request merged**